test/dom-model/DOMModelTest.js

```diff
@@ -349,6 +349,41 @@ describe("DOMModel", () => {
         })
     });
 
+    describe("byContentVal", () => {
+
+        class ButtonModel extends DOMModel {
+            @byAttrVal() variant;
+            @byContentVal() label;
+		}
+		
+        beforeEach(() => {
+            let result = makeModel(ButtonModel, `
+                <my-button variant='action'>Push Me</my-button>
+            `);
+            element = result.element;
+            model = result.model;
+		});
+		
+		it("parses the model", () => {
+            expect(model.variant).to.equal('action');
```

Comment:
Can we fix these problems with the alignment? I will have to put linting on this repo.

Reply:
Fixed. Sorry about that!